test/functional/3rd-party-repo/3rd-party-repo-remove-negative.bats

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup(){
+
+	create_test_environment "$TEST_NAME"
+
+	contents=$(cat <<- EOM
+		[test1]
+		url=www.abc.com
+		\n
+		[test2]
+		url=www.efg.com
+		\n
+		[test4]
+		url=www.pqr.com
+		\n
+		[test5]
+		url=www.lmn.com
+		\n
+	EOM
+	)
+
+	repo_config_file="$STATEDIR"/3rd_party/repo.ini
+	run sudo sh -c "mkdir -p $STATEDIR/3rd_party/"
+	run sudo sh -c "mkdir -p $PATH_PREFIX/opt/3rd-party/{test1,test2,test3,test4,test5}"
+	write_to_protected_file -a "$repo_config_file" "$contents"
+
+}
+
+test_teardown(){
+
+	destroy_test_environment "$TEST_NAME"
+
+}
+
+@test "TPR009: Negative test, repo remove usage" {
+
+	run sudo sh -c "$SWUPD 3rd-party remove $SWUPD_OPTS"
+	assert_status_is "$SWUPD_INVALID_OPTION"
+	expected_output=$(cat <<-EOM
+			Error: The positional args: repo-name is missing
+			*.*
+	EOM
+	)
+	assert_regex_in_output "$expected_output"
```

Comment:
wouldn't it look cleaner that instead of using "assert_regex_in_output" you use "assert_in_output" i these tests and you can remove the *.* part?
I don't see the point of using a regex to check the output when you are anyway only using _in_ to check for only one deterministic line within the output.